CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ All notable changes to [camunda-bpmn-js](https://github.com/camunda/camunda-bpmn
 
 ___Note:__ Yet to be released changes appear here._
 
+### Key Changes in Element Templates
```

Comment:
Might as well add the version now so it can be released later 🙂

Reply:
I was not sure if there is anything else outstanding (so it could either be patch/minor release)

Reply:
Seems like we are good to go on other fronts, so I'll add a version number

Reply:
Maybe we could also include the ````DEPS`: update to `bpmn-js-element-templates@1.9.1```` portion in the changes